Softeq.XToolkit.WhiteLabel/Bootstrapper/Containers/DryIocContainerBuilder.cs

```diff
@@ -25,10 +25,19 @@ public class DryIocContainerBuilder : IContainerBuilder
         /// </summary>
         public DryIocContainerBuilder()
         {
-            _dryContainer = new Container();
+            _dryContainer = new Container(CreateContainerRules());
```

Comment:
Maybe it's better to pass rules as an (optional) parameter to constructor? This way it would require only one override to change the rules (`BootstrapperBase`) instead of two overrides (`BootstrapperBase` and `DryIocContainerBuilder`)

Reply:
yeah, I thought about this in the first commit https://github.com/Softeq/XToolkit.WhiteLabel/pull/496/commits/7c44ac3bc91a7a0fec02271d80433dd3fc56614c 
also, I added new changes, pls review